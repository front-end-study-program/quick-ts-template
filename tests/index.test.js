import { test, expect } from 'vitest';
import { add } from '@/utils/add';
test('add.ts', () => {
    expect(add(1, 2)).toMatchSnapshot(3);
});
//# sourceMappingURL=index.test.js.map