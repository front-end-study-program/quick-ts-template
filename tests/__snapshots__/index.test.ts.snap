// Vitest Snapshot v1

exports[`add.ts 1`] = `4`;

exports[`index.ts 1`] = `3`;
